### PR TITLE
fix: typos and grammatical errors in intro.json

### PR DESCRIPTION
**Pull request by ErickOhm**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

### CODE REVIEW

**Review by ojeytonwilliams — DISMISSED** [review]
LGTM 👍

**Review by Nitesh-thapliyal — DISMISSED** [review]
LGTM👍

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -432,7 +432,7 @@
     "title": "Machine Learning with Python",
     "intro": [
       "Machine learning has many practical applications that you can use in your projects or on the job.",
-      "In the Machine Learning with Python Certification, you'll use the TensorFlow framework to build several neural networks. A you'll explore more advanced techniques like natural language processing and reinforcement learning.",
+      "In the Machine Learning with Python Certification, you'll use the TensorFlow framework to build several neural networks, and you'll explore more advanced techniques like natural language processing and reinforcement learning.",
```
Comment (RandellDawson) [inline client/i18n/locales/english/intro.json:435]:
```suggestion
      "In the Machine Learning with Python Certification, you'll use the TensorFlow framework to build several neural networks, and explore more advanced techniques like natural language processing and reinforcement learning.",
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@erickmack See my one suggestion.

**Comment by erickmack** [comment]
> 
> 
> @erickmack See my one suggestion.

I like it 👍

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by raisedadead** [comment]
Hi @RandellDawson this is pending your review. Just a gentle reminder if you are happy with this.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@erickmack Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**